src/gui/sdlmain.cpp

```diff
@@ -251,6 +251,55 @@ enum PRIORITY_LEVELS {
 	PRIORITY_LEVEL_HIGHEST
 };
 
+class pp_scale_t {
```

Comment:
I usually associate names like `some_thing_t` as a typedef, not class names.

Perhaps `pp_scale` (or `PPScale`) would be more appropriate.

Reply:
Switched.